src/cmd-sign

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/python3
+
+import argparse
+import gi
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+import tarfile
+import time
+
+import botocore
+import boto3
+
+from fedora_messaging import message
+from fedora_messaging.api import publish
+from fedora_messaging.config import conf
```

Comment:
I'm not opposed to this but...we may consider trying to have e.g. FCOS create a derived `cosa-fcos-pipeline` that includes this type of additional stuff that wouldn't necessarily be needed by others.

(Also relatedly I've been thinking about a `cosa-rpm-ostree` container or so that would mostly be rpm-ostree, and have `cosa-imagebuilder` derive from it, then you could imagine our baseline `cosa` which has the cloud stuff, then `cosa-fcos` would have that plus fedmsg...)

Reply:
Yeah, I think we do want derived containers eventually so we can include things from https://github.com/coreos/fedora-coreos-releng-automation/ too. On the RHCOS side at least, we could probably also split out `koji-upload`.

At the same time, this code is pretty integrated into the implementations of cosa and workdir management. Here for example, we're injecting back the commitmeta object into the `ostree-commit.tar` file, then also injecting it into the `tmp/repo` OSTree repo so `buildextend-[qemu|metal]` can pick it up. Keeping the code itself as part of cosa would mean e.g. refactors, cmdlib changes, etc... happen in a single repo/PR.

One thought I had about this is to have "features" like in cargo where we define env vars in the OCP BuildConfig so that conditional deps needed for say, signing, get installed as well. This would be more of a "rebuild" than a "derive" though. (Could clearly do both of course).